llvm/lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -2474,6 +2474,45 @@ SDValue SelectionDAG::getPartialReduceAdd(SDLoc DL, EVT ReducedTy, SDValue Op1,
   return Subvectors[0];
 }
 
+/// Given a store node \p StoreNode, return true if it is safe to fold that node
+/// into \p FPNode, which expands to a library call with output pointers.
+static bool CanFoldStoreIntoFPLibCall(StoreSDNode *StoreNode, SDNode *FPNode) {
```

Comment:
Start with lowercase. This also has nothing to do with FP libcalls, it's a libcall with one or more out arguments